CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ## [Unreleased] - yyyy-mm-dd
 
 ### Added
-* Added handling the OnTransportFailure callback (#707)
+* Added handling the OnTransportFailure callback (#707). This callback is invoked when a failure happens on the transport's side. Boss Room now handles that callback by returning to the Offline state.
```

Comment:
```suggestion
* Added handling the OnTransportFailure callback (#707). This callback is invoked when a failure happens on the transport's side, for example if the host loses connection to the Relay service. This won't get called when the host is just listening with direct IP, this would need to be handled differently (by pinging an external service like google to test for internet connectivity for example). Boss Room now handles that callback by returning to the Offline state.
```